Project-Lifecycle.md

```diff
@@ -98,6 +98,8 @@ vote to abstain from representation on the TSC.
 
 ## Applying to join
 
+The Node.js Foundation only admits projects written in Node.js.
```

Comment:
Node.js, libuv, http-parser are all not "written in Node.js" (whatever that means).